.ci/infra/terraform/main.tf

```diff
@@ -168,6 +168,7 @@ module "project-services" {
     "apikeys.googleapis.com",
     "appengine.googleapis.com",
     "appengineflex.googleapis.com",
+    "apphub.googleapis.com",
```

Comment:
please remove this like in [`aa5bcff` (#10079)](https://github.com/GoogleCloudPlatform/magic-modules/pull/10079/commits/aa5bcffc499464b874800b275c579c1817482478), it is stopping the build

Reply:
Removed.